site/content/release-guide.md

```diff
@@ -188,18 +198,55 @@ cd polaris-dist-dev
 mkdir x.y.z
 cp /path/to/polaris/github/clone/repo/build/distribution/* x.y.z
 cp /path/to/polaris/github/clone/repo/runtime/distribution/build/distributions/* x.y.z
-cp -r /path/to/polaris/github/clone/repo/helm/polaris helm-chart/x.y.z 
 svn add x.y.z
+```
+
+### Stage Helm Chart package
+
+You can now create a Helm package with the following command in the Polaris source folder:
+
+```
+cd helm
+helm package --sign --key '<YOUR KEY ID HERE>' --keyring ~/.gnupg/secring.kbx polaris
```

Comment:
the key id didn't work for me, I have to use the name like this: 
```
helm package helm/polaris \
  --destination build/helm \
  --sign \
  --key 'Yufei Gu' \
  --keyring ~/.gnupg/helm-secring.gpg
```

Reply:
I think it's also worth to mention how key ring is generated

Reply:
How far do we want to go on the gpg-setup side of things?  Like, should we assume (from the prerequisites) that a maintainer has properly configured and published their GPG keys?  Or should we include a howto in this guide?

Personally, I would keep this guide as focused as possible on the release and assume it is the responsibility of the maintainer to have a properly configured identity.  Which means that we could replace this command by:

```suggestion
helm package polaris
helm gpg sign polaris-x.y.z.tgz
```

wdyt?

Reply:
The GPG key generation guide (https://infra.apache.org/release-signing.html#generate) is pretty good, we may just ensure people to follow the instructions strictly in this guide. There is no need to add extra information if they are covered already.